app/conf/als-example.conf

```diff
@@ -21,6 +21,9 @@ hdfs-base = "hdfs:///user/example/Oryx"
 
 oryx {
   id = "ALSExample"
+  als {
+    rescorer-provider-class = null
```

Comment:
That's fine. The purpose here is maybe just to highlight the existence of this parameter rather than really use it.